sdk/mixedreality/Microsoft.Azure.Management.MixedReality/src/Microsoft.Azure.Management.MixedReality.csproj

```
@@ -6,18 +6,23 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.MixedReality</PackageId>
     <Description>Microsoft Azure Management Library of Mixed Reality cloud services.</Description>
-    <Version>0.9.0-preview.1</Version>
+    <Version>1.0.0.0</Version>
```

```suggestion
    <Version>1.0.0</Version>
```
3-digit versions are used in csproj and 4-digit is in AssemblyInfo.

---

Thank you very much for the helping, @isra-fel Yeming.